src/plugins/intel_gpu/src/kernel_selector/kernels/permute/permute_kernel_f_y_axes.h

```diff
@@ -23,7 +23,7 @@ class PermuteKernel_f_y_axes : public PermuteKernelBase {
     JitConstants GetJitConstants(const permute_params& params, const CommonDispatchData& dispatchData) const override;
     CommonDispatchData SetDefault(const permute_params& params) const override;
     std::vector<FusedOpType> GetSupportedFusedOps() const override {
-        return {FusedOpType::ACTIVATION, FusedOpType::QUANTIZE, FusedOpType::ELTWISE};
+        return {FusedOpType::ACTIVATION};
```

Comment:
Fusing is basic and frequently happening, could I ask to make it support fusing cases too, instead of just disabiling?

Reply:
Fusing was disabled intentionally. Adding fusing to this operation may lead to performance degradation since reading of the second input of fused operation is not coalesced while the main idea of permute optimization is to avoid uncoalesced memory access.

Reply:
Only activation fusing kept since it doesn't require second argument.

Reply:
@yeonbok After closer look at the optimized permute kernel I figured out that eltwise fusing is mandatory for this kernel, otherwise it's not selected. I'll add fusing support and update this PR.